docs/how-tos/trigger-sync.md

```diff
@@ -28,7 +28,7 @@ You can use it as the basis of your own custom logic.
 
 {{% alert title="In this page" color="tip" %}}
 
-1. [Use the `CreateShouldSyncReading` function to enable sync](#use-the-createshouldsyncreading-function-to-enable-sync)
+1. [Return `should_sync` as a reading from a sensor](#return-should_sync-as-a-reading-from-a-sensor)
```

Comment:
If builtin data manager is configured with a sync sensor, data manager will only sync on the sync interval if  the syncs sensor's `Readings` method returns a successful response and that response has a `"should_sync"` key 
in it with a value of `true`.

Otherwise, a builtin data manager will not sync on the sync interval.

Reply:
I've adjusted the descriptions below a bit - what you commented on is really only a table of contents - so not really a place to explain this further.